turbo.json

```
@@ -16,6 +16,11 @@
     "dev": {
       "cache": false,
       "persistent": true
+    },
+    "start": {
+      "dependsOn": ["^build"],
```

```suggestion
      "dependsOn": ["build"],
```

가 맞지 않나 싶은데, ^ 으로 적으신 이유가 있으신가요?

---

놓쳤어요.! [72ae41a](https://github.com/hamsurang/origin/pull/36/commits/72ae41ad43d7f8f002b9ccb58ce0ae428b4adccd)에서 반영 완료!